common/src/main/java/tachyon/Constants.java

```diff
@@ -152,6 +159,9 @@
   public static final String ZOOKEEPER_LEADER_PATH = "tachyon.zookeeper.leader.path";
   public static final String ZOOKEEPER_LEADER_INQUIRY_RETRY_COUNT =
       "tachyon.zookeeper.leader.inquiry.retry";
+  public static final String KEYVALUE_ENABLED = "tachyon.keyvalue.enabled";
```

Comment:
KEYVALUE is inconsistent with KEY_VALUE_WORKER_CLIENT_SERVICE_NAME and KEY_VALUE_MASTER_NAME